wrangler-core/src/test/java/io/cdap/directives/parser/JsPathTest.java

```diff
@@ -70,16 +70,17 @@ public void testJSONFunctions() throws Exception {
     );
 
     String[] directives = new String[] {
-      "set-column s0 json:select(body, true, \"$.name.fname\", \"$.name.lname\")",
-      "set-column s1 json:select(body, true, \"$.name.fname\")",
-      "set-column s11 json:select(body, true, \"$.numbers\")",
-      "set-column s2 json:select(body, true, \"$.numbers\")",
-      "set-column s4 json:drop(body, \"numbers\", \"integer\", \"float\", \"aliases\", \"name\")",
-      "set-column s5 json:join(s11, \":\")"
+      "set-column body json:Parse(body)",
+      "set-column s0 json:Select(body, '$.name.fname', '$.name.lname')",
+      "set-column s1 json:Select(body, '$.name.fname')",
+      "set-column s11 json:Select(body, '$.numbers')",
+      "set-column s2 json:Select(body, '$.numbers')",
+      "set-column s6 json:ArrayLength(json:Select(body, '$.numbers'))",
```

Comment:
why are these changes needed? Does this mean whoever is using wrangler will have to reapply `set-column` with new syntax?

Reply:
They were broken, so I don't even know if anyone was using them.